src/pb_memory.cc

```diff
@@ -297,9 +297,13 @@ PbMemory::GetGPUStartAddress()
     CUDAHandler& cuda_api = CUDAHandler::getInstance();
     CUdeviceptr start_address;
```

Comment:
Shouldn't we initialize `start_address` to some value before returning it if data_ptr_ is null?

Reply:
Fixed.